apps/condo/domains/organization/tasks/migrateOrganizationTinField.js

```diff
@@ -0,0 +1,106 @@
+const { getSchemaCtx } = require('@core/keystone/schema')
```

Comment:
it's one time task. It's better to move all this logic inside the bin. You don't need to reuse it

Reply:
Moved